libshvvisu/src/timeline/graph.cpp

```diff
@@ -466,24 +466,27 @@ QString Graph::prettyBitFieldValue(const QVariant &value, const TypeDescr &type_
 	return text;
 }
 
-QMap<QString, QString> Graph::prettyMapValue(const QVariant &value, const TypeDescr &type_descr) const
+QMap<QString, QString> Graph::prettyMapValue(const QVariant &value, const shv::core::utils::ShvLogTypeDescr &type_descr) const
 {
 	QMap<QString, QString> ret;
 	const QVariantMap &map = value.toMap();
 	for (auto it = map.cbegin(); it != map.cend(); ++it) {
 		QString value = it.value().toString();
 		for (auto &field : type_descr.fields) {
-			if (QString::fromStdString(field.name) == it.key() && field.typeDescr.type == shv::core::utils::ShvLogTypeDescr::Type::Enum) {
-				value = model()->typeDescrFieldName(field.typeDescr, it.value().toInt());
-				break;
+			if (QString::fromStdString(field.name) == it.key()) {
+				const shv::core::utils::ShvLogTypeDescr &field_type_descr = model()->typeInfo().types.at(field.typeName);
```

Comment:
Martine, delas to porad, co kdyz ten klic neexistuje, kdo odchyti tu vyjjimku? https://en.cppreference.com/w/cpp/container/map/at